es/13_1_Escribiendo_Puzzle_Scripts.md

```diff
@@ -0,0 +1,523 @@
+# 13.1: Escribir guiones de rompecabezas
+
+Los scripts de Bitcoin realmente _no_ tienen que depender del conocimiento de una clave secreta. En cambio, pueden ser rompecabezas de cualquier tipo.
+
+## Escribir guiones de álgebra simples
+
+Nuestro primer script real, de [§9.2: Ejecución de un script de Bitcoin](09_2_Ejecutando_un_Script_Bitcoin.md) fue un acertijo alegbraico. Ese script de Bitcoin, `OP_ADD 99 OP_EQUAL`, podría haberse descrito alternativamente como` x + y = 99`.
```

Comment:
- `[§9.2: Ejecutando un script de Bitcoin]`. Possible change needed in #377.
- **Correct** `...fue un acertijo alegbraico...` **with** `...fue un acertijo algebraico...`